dev/PushNotifications/PushNotifications-Constants.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// Rely on _STRINGIZE(x) in yvals_core.h
+#ifndef _STRINGIZE
+#define _STRINGIZEX(x)  #x
+#define _STRINGIZE(x)   _STRINGIZEX(x)
+#endif
+
+#define PR_PUSHNOTIFICATIONS_LIBID_UUID          CE96C745-3017-460E-895B-4FD98E1194F2
```

Comment:
Project Reunion is dead! Long line WindowsAppSDK! :P

I'm working on a PR to change all(!) existing references across the repo. New code should follow the new naming rules, which are pretty simple:

1. "Windows App SDK" - docs/displayname/comments/etc and where spaces are legal
2. "WindowsAppSDK" - most everywhere else

There's some other exotic rules but not worth mentioning here.

For renaming preserve your current case, e.g. PROJECTREUNION_BUILD_PIPELINE becomes WINDOWSAPPSDK_BUILD_PIPELINE etc

I'll have a PR up later today or tomorrow with renames. If you expect to checkin today or tomorrow just let me know when you do and I'll merge and update in my PR. If you don't expect to checkin until next week you'll need to rename to match my PR before you checkin

Reply:
It would be great if I can checkin today so that it will start building with the nightly build. I'll change the pipeline name like you mentioned too.

Reply:
I suggest leaving your names alone. If you do you won't build correctly until my PR comes in. I've got the change process down I can almost do it in my sleep and some things spiral widely so hard to do just some.

Shrug. Just part of the price of The Great Rename. I've handled worse :P

Reply:
Gotcha, I'll leave it alone then.

Reply:
I changed some of the variable names but left the pipeline name alone.